### ISSUE TITLE
[Feature Request]: Bulk Edit Custom Field of an Asset

### ISSUE THREAD

**Issue by That-Human-Being** [grokability/snipe-it#14853]
### Is your feature request related to a problem? Please describe.

I'm adding a field to indicate whether a given device is managed via Active Directory or LDAP, it's a basic Radio Button field, but in order to properly assign the value for over 200 devices, I have to edit each asset on at a time, the bulk edit option is incredibly limited in what I can change with it.

### Describe the solution you'd like

Basically if there is a field that all selected assets have, I would like to be able to alter said field for all selected assets

### Describe alternatives you've considered

Doing it manually, which is incredibly tedious, and I have not come across any way to automate the process

**Comment by snipe** [comment]
Bulk editing custom fields has been a feature for quite some time, so I'm not sure what the ask is. What version are you running?

**Comment by That-Human-Being** [comment]
v6.1.2, I wasn't aware that things had changed in newer releases, sorry
